src/api/middleware/errorHandler.js

```diff
@@ -0,0 +1,56 @@
+const { isCelebrate } = require("celebrate");
+const EscapeHtml = require("escape-html");
+
+const { errorExtractor } = require("../../lib/dbErrorExtractor");
+
+const ERROR_TYPES = Object.freeze({
```

Comment:
Let's rename this variable to ErrorTypes, so it can be used as ErrorTypes.FORBIDDEN, for example, I think it is more common, at least in java. (does not have to be changed here, but it should, to ensure consistency)